xbmc/video/dialogs/GUIDialogVideoInfo.cpp

```diff
@@ -740,7 +740,8 @@ class CVideoPlayActionProcessor : public CVideoPlayActionProcessorBase
                                           : ContentUtils::PlayMode::PLAY_ONLY_THIS};
 
     //! @todo get rid of special handling for movie versions
-    if (m_item.GetVideoInfoTag()->m_type == MediaTypeMovie)
+    if (m_item.GetStartOffset() != STARTOFFSET_RESUME &&
```

Comment:
Although this might work (did no runtime test) I consider this just another workaround for the actual problem that the video version dialog needs to be triggered earlier, by the implementation of the videoplayactionprosessor (base class).

Reply:
I feel this is a bandaid for only one place out of many where we also see this problem.